### DISCUSSION TITLE
How do I modify heap memory

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#630] [Q&A]
I use NoiseModelling in groovy script mode，but It has a heap memory error. How do I modify heap memory.
Additionally, I deployed noisemadelling in a container and used eclipse tmurin as the base image

**Comment by nicolas-f** [comment]
Hi

Edit wps_scripts.bat in the bin directory and edit the following line (line 36) to have:

```
set DEFAULT_JVM_OPTS=-Xmx3500m
```

**Reply by mshmoon** [reply]
My program runs on a Linux system

**Reply by mshmoon** [reply]
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/38131350/936e1b28-8947-4672-92ae-a4515d685e77)

My program runs on a Linux system

**Reply by nicolas-f** [reply]
For linux you have to edit the file you are running, {srcipts_path}

**Comment by mshmoon** [comment]
> Hi
> 
> Edit wps_scripts.bat in the bin directory and edit the following line (line 36) to have:
> 
> ```
> set DEFAULT_JVM_OPTS=-Xmx3500m
> ``` Hi,My program runs on a Linux system
